<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="userTab">
     <attribute name="title">
      <string>Client_Monitoring</string>
     </attribute>
     <widget class="QTableWidget" name="clientTableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>750</width>
        <height>205</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Uid</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> Connection Established </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> Connection Elapsed </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> Connection State </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> IP Address </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MAC Address</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="clientGruop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>440</width>
        <height>220</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Message From Client</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QTextEdit" name="clientMessage">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>421</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="serverGroup">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>300</width>
        <height>220</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Message From Server</string>
      </property>
      <widget class="QTextEdit" name="serverMessage">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>281</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sendButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>160</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="backendTab">
     <attribute name="title">
      <string>Backend_Monitoring</string>
     </attribute>
     <widget class="QTableWidget" name="socketTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>205</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Uid</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>State</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="threadTableWidget">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>10</y>
        <width>241</width>
        <height>205</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Uid</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>State</string>
       </property>
      </column>
     </widget>
     <widget class="QTextEdit" name="eventListEdit">
      <property name="geometry">
       <rect>
        <x>525</x>
        <y>10</y>
        <width>255</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>511</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
